src/handler/patch.ts

```diff
@@ -0,0 +1,67 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import logger from '../util/logger';
+import { addHttpHeaders } from '../util/httpHeaders';
+import {
+  archiveOldCreateCurrentRecord,
+  getBySystemNumberAndCreatedTimestamp,
+} from '../services/database';
+import { validateUpdateVinRequest, validateVins } from '../validators/patch';
+import { processPatchVinRequest } from '../processors/processPatchVinRequest';
+import { TechrecordGet } from '../models/post';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  logger.info('Amend VIN Called');
+
+  const isRequestInvalid: APIGatewayProxyResult | undefined = validateUpdateVinRequest(event);
+
+  if (isRequestInvalid) {
+    return isRequestInvalid;
+  }
+
+  logger.debug('Request is Valid');
```

Comment:
Have changed these to logger.debug as per Nathans suggestion so they can be toggled